### PR TITLE
Skip reviewers check on draft synchronize events

### PR DESCRIPTION
**Pull request by xacrimon**

### CODE REVIEW

**Comment by zmb3** [comment]
Will reviewers be added when the mm pull request is promoted out of draft status?

File: .github/workflows/assign.yaml [hunk 1]
```diff
@@ -25,6 +25,7 @@ permissions:
     statuses: none
 jobs:
   auto-request-review:
+    if: !github.event.pull_request.draft
```
Comment (quinqu) [inline .github/workflows/assign.yaml:28]:
```suggestion
    if: ${{ !github.event.pull_request.draft }}
```

File: .github/workflows/check.yaml [hunk 2]
```diff
@@ -28,6 +28,7 @@ permissions:
 
 jobs: 
   check-reviews:
+    if: !github.event.pull_request.draft
```
Comment (quinqu) [inline .github/workflows/check.yaml:31]:
```suggestion
   if: ${{ !github.event.pull_request.draft }}
```

**Review by quinqu — CHANGES_REQUESTED** [review]

**Comment by quinqu** [comment]
@zmb3 Yes

**Comment by russjones** [comment]
@quinqu Is there a way we can push this logic into the bot code itself? Or does it have to be in the workflow.

It probably has to be in the workflow, but maybe you know some trick?

**Comment by xacrimon** [comment]
@russjones as far as I know, this has the effect of just marking the run as done, something similar could be achieved by just skipping the check on draft PR's but we'd still need to get that info to the bot.

**Comment by quinqu** [comment]
@russjones There isn't a way to push the logic to the bot code, unfortunately.

**Comment by quinqu** [comment]
@xacrimon This change doesn't mark the run as done, it just skips it altogether.

**Comment by russjones** [comment]
Let's close this for now, we can come back to this item later when we have a better idea on how to address it.

**Comment by quinqu** [comment]
@russjones This change works, it has been addressed.

**Comment by quinqu** [comment]
@russjones @xacrimon I am just going to make this change since this PR had been open for a while.

**Pull request closed without merging**